integration-tests/src/common/java/pl/allegro/tech/hermes/integrationtests/setup/HermesConsumersTestApp.java

```diff
@@ -26,10 +30,13 @@ public HermesTestApp start() {
                 "--consumer.healthCheckPort=0",
                 "--consumer.kafka.namespace=itTest",
                 "--consumer.kafka.clusters.[0].brokerList=" + kafka.getBootstrapServersForExternalClients(),
+                "--consumer.kafka.clusters.[0].clusterName=" + "primary-dc",
```

Comment:
I see that in management configuration we have following configuration:
`kafka.clusters[0].clusterName=primary` 
Won't it be a problem that there are two different values for management/consumers?

Reply:
Actually final configuration is the same, because management is building qualifiedClusterName with suffix containing dc name.